### PR TITLE
Add pagination to package_jobs API endpoint

### PR DESCRIPTION
**Pull request by raskchanky**
tenor-190529131

Signed-off-by: Josh Black <raskchanky@gmail.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-jobsrv/src/data_store.rs [hunk 1]
```diff
@@ -416,12 +431,20 @@ impl DataStore {
...
+                &(project.get_start() as i64),
+            ],
         ).map_err(Error::ProjectJobsGet)?;
 
-        let mut response = jobsrv::ProjectJobsGetResponse::new();
         let mut jobs = protobuf::RepeatedField::new();
+        let mut response = jobsrv::ProjectJobsGetResponse::new();
+        response.set_start(project.get_start());
+        response.set_stop(self.last_index(project, &rows));
+        response.set_count(rows.len() as u64);
```
Comment (mwrock) [inline components/builder-jobsrv/src/data_store.rs]: I thought this was intended to be the total count of all rows in the db table but it looks like this is just returning the number of rows in this result set.

Comment (raskchanky) [reply components/builder-jobsrv/src/data_store.rs]: Ah yes, my bad.  I'll fix that up.

**Comment by raskchanky** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/274392331).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/274392331).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**